lua/nvim-tree/popup-menu/init.lua

```diff
@@ -0,0 +1,79 @@
+local a = vim.api
+
+local M = {}
+
+local function get_node()
+  local get_node_at_cursor = require'nvim-tree.lib'.get_node_at_cursor
+  
+  local current_node = get_node_at_cursor() 
+  return current_node
+end
+
+local current_popup = nil
```

Comment:
This seems to be global. Please leave a TODO indicating that we need to test / fix this when we are using multiple tabs.